site/content/docs/user/using-wsl2.md

```diff
@@ -84,6 +84,47 @@ nodes:
 1. create service `kubectl create service nodeport nginx --tcp=80:80 --node-port=30000`
 1. access service `curl localhost:30000`
 
+## Kubernetes Service with Session Affinity
```

Comment:
@BenTheElder should we start a "Known issues" section here, and nest this inside it?

Reply:
This is fine for now, we can change it later, I'm happy to have something covering this, thank you!